src/vivarium_testing_utils/automated_validation/visualization/dataframe_utils.py

```diff
@@ -110,7 +107,14 @@ def format_draws_sample(draw_list: list[int], source: DataSource) -> str:
     draw_list = sorted(draw_list)
     if source == DataSource.SIM:
         # Display all draws run in the simulation
-        return str(draw_list)
+        if len(draw_list) < 6:
+            return str(draw_list)
+        else:
+            # Display 5 draws per line
+            lines = []
+            for i in range(0, len(draw_list), 5):
+                lines.append(str(draw_list[i : i + 5]))
+            return "\n".join(lines)
```

Comment:
If you join with `"<br>"` here, you shouldn't need to do the `replace()` in the interface method.